lib/runtime/wasmer/instance.go

```diff
@@ -100,6 +102,15 @@ func NewInstance(code []byte, cfg *Config) (*Instance, error) {
 	return newInstance(code, cfg)
 }
 
+// FuncCallerFileHandler returns a Handler that adds the name of the calling function to the context with key "func"
+//  and the line number and file of the calling function to the context with key "caller".
+func FuncCallerFileHandler(h log.Handler) log.Handler {
```

Comment:
Maybe rename to `customFileHandler` and make the function private.

Reply:
can we put this in the `runtime` package? since the other wasm interpreters will likely end up using the same logger

Reply:
Moved to `runtime` package, renamed to `CustomFileHandler`, kept public since it was moved to runtime package.